svm/src/loader.rs

```diff
@@ -3,12 +3,14 @@ use {
     solana_sdk::{account::AccountSharedData, pubkey::Pubkey},
 };
 
-/// Runtime callbacks for transaction processing.
-pub trait TransactionProcessingCallback {
-    fn account_matches_owners(&self, account: &Pubkey, owners: &[Pubkey]) -> Option<usize>;
-
+/// The "loader" required by the transaction batch processor, responsible
+/// mainly for loading accounts.
+pub trait Loader {
+    /// Load the account at the provided address.
     fn get_account_shared_data(&self, pubkey: &Pubkey) -> Option<AccountSharedData>;
 
+    fn account_matches_owners(&self, account: &Pubkey, owners: &[Pubkey]) -> Option<usize>;
```

Comment:
This function will likely go away with the rework in accounts-db. Adding it to this trait would cause extra maintenance later on for the user of SVM.

Reply:
Technically it was already there.

Reply:
Yes, you are right. Ignore this comment.